# Used to broadcast the current mode to tell which nodes to activate



char mode
---
bool success
